### DISCUSSION TITLE
聚会

### DISCUSSION THREAD

**Discussion by langningchen** [XMOJ-Script-dev/XMOJ-Script#543] [General]
@PythonSmall-Q @boomzero XMOJ-Script开发组的各位主要成员，我们作为初三学生已经中考完了，也辛苦你们两个帮我维护这个项目了，到暑假里有空的时候我请你们一起出去吃个饭 😄 另：我的项目 XMOJ-Client 已经开始开发了，基本框架已经写好了，只要等我慢慢将解析等内容填充进去就好了

**Comment by boomzero** [comment]
@langningchen emm... 数学已经-20了...

**Reply by boomzero** [reply]
已经没有心情开发东西了 虽然也有一个在写的idea

**Reply by boomzero** [reply]
幸好别的没扣分

**Comment by boomzero** [comment]
Anyways, good luck on `XMOJ-Client`!


~Can I ask what tech stack you're using?~

**Comment by langningchen** [comment]
@boomzero 所以你有可能能来吗？我命到时候再定时间

**Reply by PythonSmall-Q** [reply]
几号呀

**Reply by boomzero** [reply]
> @boomzero 所以你有可能能来吗？我命到时候再定时间



几号呀

**Comment by langningchen** [comment]
> Can I ask what tech stack you're using?

大概后端就是 SQLite, C++, http-server, json 等等
前端用的 tsx, react, bootstrap 等等

**Reply by boomzero** [reply]
~Why C++~

**Reply by langningchen** [reply]
……

**Reply by boomzero** [reply]
Seriously, it is not safe

**Comment by PythonSmall-Q** [comment]
@langningchen 要不把xmoj-client放出来让我们看看？（做完了别忘了放在product hunt上哦

**Comment by langningchen** [comment]
@boomzero 我现在在忙着算法，那个东西最近几天没怎么搞

**Reply by boomzero** [reply]
正在准备自主招生……

**Comment by langningchen** [comment]
@PythonSmall-Q @boomzero 你们几号有空？月底前我都可以，但是只能在17点以后或7点以前，你们都住在大概哪里？我选一个都近一点的地方

**Reply by boomzero** [reply]
闵行....

**Reply by langningchen** [reply]
@PythonSmall-Q 你是在静安吗？

**Reply by boomzero** [reply]
> @PythonSmall-Q 你是在静安吗？

Probably Huangpu district

**Reply by langningchen** [reply]
@boomzero 那也差不多，徐汇区应该可以

**Reply by PythonSmall-Q** [reply]
我黄埔

**Comment by langningchen** [comment]
要么我们去徐汇区（是不是在这种公开讨论里说这种事情不太好）

**Reply by boomzero** [reply]
~lets go to Ittoqqortoormiit~

**Comment by langningchen** [comment]
@PythonSmall-Q @boomzero 那就徐汇区晚上18:30，在哪天你们有空？你们有什么推荐的餐厅吗？

**Reply by PythonSmall-Q** [reply]
几号?我看看时间

**Comment by boomzero** [comment]
emm... maybe we shouldn't discuss this on Github?

**Comment by langningchen** [comment]
@boomzero 我也是这样认为的，主要原因就是你没有微信，我和 @PythonSmall-Q 可以在微信上说

**Comment by langningchen** [comment]
@PythonSmall-Q 应该是这周六 22号 晚上16:30

**Comment by boomzero** [comment]
emm… This Saturday?Sent from my iPadOn Jun 20, 2024, at 08:20, Langning Chen ***@***.***> wrote:﻿
@PythonSmall-Q 应该是这周六

**Comment by langningchen** [comment]
@boomzero 是的，行吗？

**Comment by boomzero** [comment]
Sorry, 学校要排练….

**Comment by langningchen** [comment]
@boomzero 那这周日？

**Comment by boomzero** [comment]
Wait a sec, Where?

**Comment by PythonSmall-Q** [comment]
@langningchen 这个周末我没空。PS：7/5前（含当天）都没空，然后所有的周六从早到晚全是课

**Reply by PythonSmall-Q** [reply]
抱歉哈，真的很忙

**Comment by langningchen** [comment]
好吧，那估计聚不了了，因为我7/1之后都没空

**Comment by langningchen** [comment]
或者我再看一下

**Comment by langningchen** [comment]
我一点高中的东西都还没学呢，你们太卷了

**Comment by boomzero** [comment]
自招?

**Reply by langningchen** [reply]
也不是

**Comment by langningchen** [comment]
[Chen Langning Calendar.ics.txt](https://github.com/user-attachments/files/15916513/Chen.Langning.Calendar.ics.txt)

这是我暑假里目前安排的日历，你们可以看一下什么时候能出去吃 @PythonSmall-Q @boomzero 
**注意：为了上传文件，我增加了扩展名 `.txt` ，请在查看时删除 `.txt`**

**Comment by boomzero** [comment]
怎么打开啊...

**Comment by boomzero** [comment]
我一打开它就让我import calender

**Comment by boomzero** [comment]
Ahhh import 了

**Comment by boomzero** [comment]
怎么撤销啊

**Comment by langningchen** [comment]
…… @boomzero

**Comment by langningchen** [comment]
@boomzero 你看邮箱了吗……

**Comment by langningchen** [comment]
哦，你看了……

**Comment by langningchen** [comment]
@boomzero 具体信息邮件发你了

**Comment by langningchen** [comment]
@boomzero 你为什么又没有看邮箱（
这样交流太慢了， @PythonSmall-Q 想要让你注册一个微信
或者你邮件回的稍微及时一点

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 很抱歉，因为我不再在 XMOJ 上课，所以我的 XMOJ-Client 已经正式转变为 CYEZOJ-Client 项目了，因为他们都是基于 HustOJ 的而且我的项目还才只写了框架，所以我迁移比较方便。我再次在此致歉
